src/functions/public/Get-ContextVault.ps1

```diff
@@ -0,0 +1,108 @@
+﻿function Get-ContextVault {
+    <#
+        .SYNOPSIS
+        Lists all available context vaults.
+
+        .DESCRIPTION
+        Retrieves information about all configured context vaults,
+        including their names, paths, and creation dates.
+        Shows which vault is currently set as the default.
+
+        .PARAMETER Name
+        Optional name filter to get information about a specific vault.
```

Comment:
In general, I do not want parameter documentation here. I want it above the parameter specification in the param() block.